distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -2909,11 +2909,8 @@ Recommendations for Framework Development
 ===== Console Commands
 
 ${branding} supports development of custom console commands.
-For more information, see the Karaf website on http://karaf.apache.org/manual/latest-2.2.x/developers-guide/extending-console.html[Extending the Console]. 
 
-===== Custom ${branding} Console Commands
-
-${branding} includes custom commands for working with the Catalog, as described in the Console Commands section.
+${branding} includes custom commands for working with the Catalog, as described in the <<Console Commands>> section.
```

Comment:
I will follow the link and find out. It's a line that's been here forever, I just added the link now. I'll remove if it isn't helpful.